### ISSUE TITLE
上采样这儿是不是核大小应该为3？

### ISSUE THREAD

**Issue by ReckonerInheritor** [JaveyWang/PAN-pytorch#12]
self.conv_upsample_2 = nn.ConvTranspose2d(channels_mid, channels_mid, kernel_size=4, stride=2, padding=1, bias=False)
改为：
self.conv_upsample_2 = nn.ConvTranspose2d(channels_mid, channels_mid, kernel_size=3, stride=2, padding=1, bias=False)
这样修改正确吗？
